src/components/AdminSettings.vue

```diff
@@ -391,6 +391,25 @@
 			</div>
 		</div>
 
+		<div v-if="isSetup" id="esignature-settings" class="section">
+			<h2>{{ t('richdocuments', 'Electronic signature settings') }}</h2>
+			<SettingsInputText v-model="settings.esignature_base_url"
+				:label="t('richdocuments', 'URL for the electronic signature API')"
+				:hint="t('richdocuments', 'The production API URL is https://id.eideasy.com, the test API URL is https://test.eideasy.com.')"
```

Comment:
Not a blocker, just curious, apart from development is there any reason to configure the test API URL in productive usage? If not I'd remove the input for this one and rather document how to set this value with the CLI for developers.

Reply:
Yes, this is basically for development only, you're right. Where to document this, is docs/app_settings.md a good place to mention it?

Reply:
Yes, that sounds good, there might be more undocumented ones, but that is the best place from my perspective 👍

Reply:
OK, let me see, I think by now this is the only outstanding part of this PR.